### DISCUSSION TITLE
Falha Autenticação Flutter/Google ""

### DISCUSSION THREAD

**Discussion by Br-Unknown** [Flutterando/forum#157] [Perguntas e Respostas]
Galera, venho tendo dificuldades em realizar a autenticação Google no meu app, fiz todos passos que eles pedem porém quando tento realizar o login com uma conta ele não procede para a tela inicial o erro é: 

PlatformException(sign_in_failed, com.google.android.gms.common.api.ApiException: 12500: , null, null)

Alguém ai pode me dar uma Luz?

**Comment by alanleonidas** [comment]
Olá @Br-Unknown tudo bem?

Você consegue fazer a autenticação via signInWithEmailAndPassword?

**Comment by bphd** [comment]
E/Finsky  (): [173] uks.a(849): Encountered exception whilst fetching from GMS, deferring to Finsky: com.google.android.gms.common.api.ApiException: 17: API: IntegrityApiDisplayListener.API is not available on this device. Connection failed with: ConnectionResult{statusCode=API_DISABLED, resolution=null, message=null}
